cmd/kubelet/app/options/container_runtime.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	// When these values are updated, also update test/e2e/framework/util.go
```

Comment:
maybe update this comment while you're here, since it's no longer valid? :)

Reply:
Path updated, technical debt repaid. :)